content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -249,6 +247,9 @@ For a reference to old feature gates that are removed, please refer to
 | `ComponentSLIs` | `true` | Beta | 1.27 | 1.28|
 | `ComponentSLIs` | `true` | GA | 1.29 | - |
 | `ConsistentHTTPGetHandlers` | `true` | GA | 1.25 | - |
+| `CustomResourceValidationExpressions`       | `false` | Alpha | 1.23  | 1.24  |
+| `CustomResourceValidationExpressions`       | `true`  | Beta  | 1.25  | 1.26  | 1.27  | 1.28  |
+| `CustomResourceValidationExpressions`       | `true`  | GA    | 1.29  | -     |
```

Comment:
```suggestion
| `CustomResourceValidationExpressions` | `true` | GA | 1.29 | - |
```

Reply:
I have updated the changes according to the suggested format. Please feel free to take a look when have time. Thanks